app.yaml

```diff
@@ -0,0 +1,7 @@
+runtime: python27
+api_version: 1
+threadsafe: false 
+
+handlers:
+- url: /.*
+  script: app/server.py
```

Comment:
You might run into issues running `server.py` from the root directory.  In Heroku, I had to do something like this for the last project:

`cd app && python server.py`